designcompose/src/main/java/com/android/designcompose/DesignView.kt

```diff
@@ -952,6 +953,13 @@ fun DesignDoc(
     endSection()
 }
 
+// Enable the "Squoosh" refactor. Squoosh implements its own view tree, rather than using Compose,
+// which brings some performance and flexibility benefits. Squoosh isn't feature complete (no
+// scrolling, no lists, no transformed input), but it does add animations and is likely the
+// direction that DesignCompose will move in to be lighter weight and better integrate with
+// external layout.
+private const val USE_SQUOOSH = false
```

Comment:
Since this is hard coded it means that we can't run tests in CI, correct?

Reply:
Yeah, I should have written this in the PR, but I thought we could:

1. Land the change so that we get build coverage, and don't need to rebase again.
2. Add the ability to turn it on dynamically, enable test cases where it's working...
3. Iterate on fixing the implementation and enabling more test cases.